ajax/libs/turndown/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "turndown",
+  "filename": "turndown.min.js",
+  "description": "A library that converts HTML to Markdown",
+  "version": "4.0.1",
+  "author": "Dom Christie",
+  "homepage": "https://domchristie.github.io/turndown/",
+  "keywords": [
+    "converter",
+    "html",
+    "markdown"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/domchristie/turndown.git"
+  },
+  "npmName": "turndown",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "turndown*"
```

Comment:
As usual, we grab all the files in `dist` & `build` as the basePath, so in this case, we have to use `**/*` as the pattern.
If some files we don't need to fetch when the basePath is `dist` or `build`, we use excluded mode.
So, would you please revise this pattern?
Many thanks.